metadatastore/test/test_commands.py

```diff
@@ -503,12 +504,28 @@ def test_double_run_stop():
     with pytest.raises(RuntimeError):
         mdsc.insert_run_stop(run_start_uid, ttime.time(),
                              uid=str(uuid.uuid4()))
+@pytest.fixture
+def prep_header():
+    testuid = str(uuid.uuid4())
+    hdr = {'time': time.time(), 'uid': testuid,
+           'tag': 'find_last_test_tag'}
+    return hdr
 
 
 def test_find_last_for_smoke():
     last, = mdsc.find_last()
 
 
+def test_find_last(prep_header):
```

Comment:
These should also take in the `mds_all` fixture so that these tests run against all three of the mds versions.